### PR TITLE
[SmartPlaces.Facilities.OntologyMapper] Add NetStandard2.0, Net7.0, and Net8.0 Support [Part 1 of 4]

### PR DESCRIPTION
**Pull request by Tyler-Angell**
### What
Add support for NetStandard2.0, Net7.0, Net8.0 and NetFramework 4.6.2

### Why
Following in the footsteps of [Dotnet](https://github.com/dotnet/runtime/blob/aeceba3f3b84e21d7dab99c45f2163756f7ba3c3/src/libraries/Microsoft.Extensions.Configuration.Binder/src/Microsoft.Extensions.Configuration.Binder.csproj#L4C5-L4C143) I've added multitargeting for the Core Current, Previous, and Minimum, netstandard2.0, and Framework minimum as a target, 

This expands the flexibility of this nuget so that consumers can target whichever .NET version they are comfortable with.

### Tested
Passed Unit Tests
Ran successfully on internal Pilot project

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -77,6 +60,8 @@ updates:
     allow:
       - dependency-name: Microsoft.SmartPlaces.Facilities.*
       - dependency-name: Mapped.*
+      - dependency-name: RealEstateCore.*
+      - dependency-name: WillowInc.*
```
Comment (stdrenchen) [inline .github/dependabot.yml:64]: Looks like unrelated changes to the description.  Want all this in the same PR?

**Review by hammar — APPROVED** [review]
Changes look good to me but GitHub auto-flagged an earlier issue that is a 30 second fix, see screenshot below:

image

File: .github/dependabot.yml (hunk 1 above)
Comment (Tyler-Angell) [reply .github/dependabot.yml:64]: Fair, I'll pull it out. Got carried away updating the builds

**Comment by Tyler-Angell** [comment]
> Changes look good to me but GitHub auto-flagged an earlier issue that is a 30 second fix, see screenshot below:
> 
> image

Thanks for noticing that. I had that change queued for a later PR, but have brought it forward to make the build gods happy.

**Review by hammar — APPROVED** [review]
